experimenter/experimenter/kinto/tests/test_tasks.py

```diff
@@ -749,6 +752,78 @@ def test_launching_experiment_live_when_record_is_in_main(self):
             ).exists()
         )
 
+    def test_launching_experiment_live_prefflips_feature(self):
+        prefflips_feature = NimbusFeatureConfigFactory.create(
+            application=NimbusExperiment.Application.DESKTOP,
+            name="prefFlips",
+            slug="prefFlips",
+        )
```

Comment:
This is a special case here right?  Like it has to be that exact slug?  Maybe we should move this to some kind of test constant somewhere?

Reply:
yes, that exact slug. its hardcoded in the application configs

Reply:
Right so should we refer to it as a constant here?